### ISSUE TITLE
Poor accessibility and usability since redesign

### ISSUE THREAD

**Issue by katlyn** [sapic/avatarcropper#25]
From commit 1f47bac2002120345d5ceee79f08432acdd988ae and forward, updates to the site design have made it less accessible and have decreased usability. There have been some improvements (such as more clear active focus rings), but overall the changes in contrast and layout have made the site more difficult to use.

## Accessibility and Contrast
At the time of writing, the application menu has this appearance:

![avatarcropper com_](https://user-images.githubusercontent.com/17418611/194270303-5087527a-5e82-4583-a86a-2a31fbb990d5.png)

Most notably, the download button has very poor contrast - [failing the WebAIM contrast checks](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=C6C60A) for both normal and large text. Additionally, the shadow effect that is overlayed on top of the button further washes the text out to have event worse contrast. At its worst, the button appears like this with a contrast ratio of [only 1.23:1](https://webaim.org/resources/contrastchecker/?fcolor=E5E4A7&bcolor=CFD307).
![image](https://user-images.githubusercontent.com/17418611/194274343-e3a035e3-fd9c-4323-aa52-84ee04529a37.png)
The save button (after rendering) suffers from a similar issue. Pre-redesign, the render button had a contrast ratio of 11.81:1.

While more difficult to quantify as there are fewer guidelines on the subject, the contrast between a toggled and non-toggled button is also very low, making it difficult to quickly distinguish between items that are active and inactive. Adding to this, the hover color is nearly identical to the actively toggled color.

Pre-redesign hover was shown with an outline and active was shown by inverting the control, making the two states clear. However, it must be acknowledged that this is still not ideal, as it's not immediately obvious which state is active and which is inactive. This same issue exists in the redesign, but is made even worse due to the similarity in the colors for active and inactive.

## Usability
One of the largest visual changes in the redesign is that the control menu changed from a floating menu to a static sidebar - taking up all vertical space for the area of the page that it occupies. This leads to less space being available for previews on narrower screens, which in turn leads to less display area for the cropping interface without overlap from previews. With the default loaded previews, a screen width of just 1024 pixels already reduces the crop view's width to just 154 pixles - barely big enough to control, let alone make precise adjustments. This issue exists regardless of display height. Pre-redesign, this largest 512x512 preview was not present, so this issue did not present itself immediate - but with the defaults being changed, it is now immediately evident on smaller window or screen sizes.

![avatarcropper com_ (1)](https://user-images.githubusercontent.com/17418611/194280199-f957113d-d82c-49b8-9e54-d6247d5b3c07.png)

## In Summary
While I do personally prefer the old design to this one, this issue isn't necessarily to say that it needs to be undone - there are just several issues that should be addressed. Work can be done to fix these as well as to address the issues that may have already been existing pre-redesign to improve the user experience.

**Comment by huhukawa** [comment]
i can say that on mobile (safari ios 14.4.2) the website doesnt work at all and gets stuck at the rounded rectangle thing that appears for some reason before asking you to choose your file :[
![EC26F18C-2A00-4DBE-BA5A-0788E06FFC44](https://user-images.githubusercontent.com/54695830/194453811-db0e43cf-dfe2-4781-ba5e-988f8696f1be.jpeg)
this only became a problem a bit after the redesign was released 😭

**Comment by TheOddball** [comment]
Hm. Didn't even know I may have accidentally added a 512x preview as default

**Comment by TheOddball** [comment]
I do agree that the `Download` button *is* a bit hard to read, this will be adjusted immediately.

**Comment by TheOddball** [comment]
Noticibility was improved on hover of download button in 888fbebc43e432892aab4d8ae839b5458820fb6e

**Comment by TheOddball** [comment]
> i can say that on mobile (safari ios 14.4.2) the website doesnt work at all and gets stuck at the rounded rectangle thing that appears for some reason before asking you to choose your file :[ ![EC26F18C-2A00-4DBE-BA5A-0788E06FFC44](https://user-images.githubusercontent.com/54695830/194453811-db0e43cf-dfe2-4781-ba5e-988f8696f1be.jpeg) this only became a problem a bit after the redesign was released 😭

Unfortunately, I do not have an iOS device to test with
